src/main/java/Duke/Command/DateOfCommand.java

```diff
@@ -0,0 +1,40 @@
+package Duke.Command;
+
+import Duke.DukeException.DukeIncompleteException;
+import Duke.DukeException.DukeSyntaxErrorException;
+import Duke.Main.TaskList;
+import Duke.Task.Deadline;
+import Duke.Task.Event;
+import Duke.Task.Task;
+
+public class DateOfCommand extends Command {
+
+    private String description;
+    private TaskList taskList;
+    public DateOfCommand(String description, TaskList taskList) {
+        super(description, taskList);
+        this.description = description;
+        this.taskList = taskList;
+    }
+
+    @Override
+    public String reply() {
+        if (description.length() == 0) {
+            throw new DukeIncompleteException();
+        }
+        try {
```

Comment:
Logical units within a block should be separated by one blank line. I noticed the same issue in several other places too.